routers/api/v1/org/repo.go

```diff
@@ -0,0 +1,46 @@
+package org
+
+import (
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/context"
+	api "code.gitea.io/sdk/gitea"
+)
+
+// ListRepos list all of an organization's repositories.
+func ListRepos(ctx *context.APIContext) {
+	// swagger:route GET /orgs/{org}/repos orgListRepos
+	//
+	//		Produces:
+	//		- application/json
+	//
+	//		Responses:
+	//			200: RepositoryList
+	//			500: error
+
+	var apiRepos []*api.Repository
+	if ctx.User != nil {
```

Comment:
I don't like quietly returning a empty list of repositories for unauthenticated users. Either we should have a 401 (like we do for `GET /repos/:owner/:reponame`), or show the unauthenticated users the organization's public repos.